extensions/amp-list/validator-amp-list.protoascii

```diff
@@ -45,6 +45,8 @@ tags: {  # <amp-list> with mandatory src and/or [src] attr
   attrs: {
     name: "auto-resize"
     value: ""
+    deprecation: "replacement-to-be-determined-at-a-later-date"
```

Comment:
@honeybadgerdontcare Any better way to have a custom warning message?

Reply:
This would cause the message to be:
`The attribute 'auto-resize in tag 'amp-list' is deprecated - use 'replacement-to-be-determined-at-a-later-date' instead.`

Not an ideal message and not sure how to make it better other than adding a deprecation_url that points to a specific issue on github explaining the situation in a way a developer would understand.

The only other way to have a different message would be to change the engine and add a new error message. That would take longer than using this mechanism.